content/community/_index.md

```
@@ -1,34 +1,82 @@
+    { url = "https://x.com/valkey_io", text = "X" },
+    { url = "#email-signup", text = "Newsletter" }
   ] }
 ]
+
+conduct = { title = "Community Conduct", description = "Help keep the Valkey community welcoming, inclusive, and respectful for everyone.", link = { url = "/code_of_conduct", text = "Read the Code of Conduct" } }
 +++
 
-# Community
-## How to Contribute with us?
-We welcome your involvement in the Valkey community! Here are several ways you can contribute:
+## Ways to contribute
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)